src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -235,6 +235,8 @@
       url: /components/header
     - title: Heading
       url: /components/heading
+    - title: Hidden
+      url: /components/hidden
```

Comment:
I think we should remove internal components from the sidebar until we have a plan for how we want to handle documentation for internal components.

```suggestion
```